pkg/client/group_mgmt_client.go

```diff
@@ -44,20 +47,27 @@ type ErrorResponse struct {
 
 // Message is an `Error` implementation as well as an implementation of the JSON API error object.
 type Message struct {
-	Code string `json:"code,omitempty"`
-	Text string `json:"text,omitempty"`
+	Code      string   `json:"code,omitempty"`
+	Text      string   `json:"text,omitempty"`
+	Severity  string   `json:"severity,omitempty"`
+	Arguments Argument `json:"arguments,omitempty"`
+}
+
+type Argument struct {
+	JobId string `json:"job_id,omitempty"`
 }
 
 // NewClient instantiates a new client to communicate with the Nimble group
-func NewClient(ipAddress, username, password, apiVersion string) (*GroupMgmtClient, error) {
+func NewClient(ipAddress, username, password, apiVersion string, waitOnJobs bool) (*GroupMgmtClient, error) {
```

Comment:
Rather than requiring the waitForJobs option to be specified here, would it make sense to set a default value and provide another function to change it?  That pattern could open the door to additional configurability.  For example the values at lines 17-21 could be overridden.

Reply:
I think for this release it is ok. please check the above comment in the same context.